submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+`E = P * t = P * t_c * n / C`
+
+Once we have the crafting recipes, fuel energy values, and the processing energy equation, we have all we need to make the conversion diagrams. Now let's look at them:
+
+## Coal vs. liquified coal
+
+The simple way to process coal is to just liquify it and immediately convert the outputs into solid fuel, as below:
+
+{% include image.html 
+
+src= 'src'
+ alt='Processing coal into solid fuel directly'
+ caption='Processing coal into solid fuel directly' 
+
+%}
+
+We find that 10 coal yields 5.75 solid fuel this way. Meanwhile, the coal processing requires energy for heating steam and running machines. We can calculate the total energy gain by subtracting all the input energy and processing energy from the output energy. Then, the percent gain is the energy gain divided by the energy obtained from directly burning the input fuel.
+
+`Energy gain = 69 - 40 - 1.5 - 2.1 - 1.365 - 0.84 - 0.21 = 22.985MJ`
+
+`Percent gain = 22.985MJ / 40MJ = 57.4625% `
+
+So we see that simply liquifying the coal gets us about 57% more energy out of it. Now, can we get out even more if we use cracking?
+
+## Adding cracking to coal liquification
+
+Cracking adds and extra step to the process, but looking at the recipes shows that it can increase solid fuel yield: Cracking heavy oil provides 30/40 or 75% as much light oil. Meanwhile, 10 light oil is enough to make 1 solid fuel rather than 20 heavy oil, meaning that it has 200% of the yield. Combining these, by cracking heavy oil we get 75% * 200% = 150% as much solid fuel as before. 
+
+However, the same cannot be said about cracking light oil. It provides 20/30 or about 66.7% as much petroleum gas, but also 20 petroleum gas is required for making solid fuel instead of 10 light oil, meaning 50% as much yield. Combining these, by cracking light oil we get 66.7% * 50% = 33.4% as much solid fuel as before, meaning that light oil cracking should be avoided.
+
+Now to look at the process with heavy oil cracking:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking'
+ caption='Processing coal into solid fuel with heavy oil cracking'
+
+%}
+
+We see that we now get 7.375 solid fuel from the same 10 coal instead of 5.75. That seems great but the cracking also consumes additional energy. So, let's look at total energy gain:
+
+`Energy gain = 88.5 - 40 - 1.5 - 2.1 - 0.6825 - 2.8875 - 0.21 = 41.12MJ`
+
+`Percent gain = 41.12MJ / 40MJ = 102.8% `
+
+We find that liquifying coal and including cracking leads getting about **twice its original energy**. In other words, if you liquify coal before burning it, you only need half as much as you did before! While this suggests that liquifying coal on a large scale is a great idea, other factors like setup costs need to be considered before jumping into it, as discussed later. 
```

Comment:
```suggestion
We find that liquifying coal and including cracking leads to getting about **twice its original energy**. In other words, if you liquify coal before burning it, you only need half as much as you did before! While this suggests that liquifying coal on a large scale is a great idea, other factors like setup costs need to be considered before jumping into it, as discussed later. 
```

Reply:
I'm not sure about the added  "the coal" here...